content/md/en/docs/tutorials/build-a-blockchain/simulate-network.md

```diff
@@ -170,15 +167,14 @@ To add a node to the running blockchain:
    --bob \
    --port 30334 \
    --rpc-port 9946 \
-   --rpc-port 9934 \
    --telemetry-url "wss://telemetry.polkadot.io/submit/ 0" \
    --validator \
    --bootnodes /ip4/127.0.0.1/tcp/30333/p2p/12D3KooWEyoppNCUx8Yx66oV9fJnriXwCcXwDDUA2kj6vnc6iDEp
    ```
 
    Note the following differences between this command and the previous one:
 
-   - Because the two nodes are running on the same physical computer, you must specify different values for the `--base-path`, `--port`, `--rpc-port`, and `--rpc-port` options.
+   - Because the two nodes are running on the same physical computer, you must specify different values for the `--base-path`, `--port`, and `--rpc-port` options.
```

Comment:
```suggestion
   - Because the two nodes are running on the same physical computer, you must specify different values for the `--base-path`, `--port` and `--rpc-port` options.
```

Reply:
sorry for the nit-pick